README.md

```diff
@@ -37,7 +37,8 @@ The donkey car is controlled by running a sequence of events
 import time
 from donkeycar import Vehicle
 from donkeycar.parts.cv import CvCam
-from donkeycar.parts.datastore_v2 import TubWriter
+# FIXME from donkeycar.parts.datastore_v2 import TubWriter => needs class TubWriter in ./datastore_v2.py
+from donkeycar.parts.datastore import TubWriter
```

Comment:
You are right, the import is not correct, but it should be from `donkeycar.parts.tub_v2` rather, can you change that and remove the #fixme, then we can merge this.

Reply:
commited